style.css

```diff
@@ -12,23 +12,27 @@ pre {
 pre code {
   white-space: inherit;
 }
-.rmdcaution, .rmdimportant, .rmdnote, .rmdtip, .rmdwarning {
+.rmdcaution,
+.rmdimportant,
+.rmdnote,
+.rmdtip,
+.rmdwarning {
   padding: 1em 1em 1em 4em;
   margin-bottom: 10px;
   background: #f5f5f5 5px center/3em no-repeat;
 }
 .rmdcaution {
-  background-image: url("images/caution.png");
+  background-image: url("images/logos/caution.png");
```

Comment:
These should be /icon/ not /logos/, right?

Reply:
Good catch.